source/query/filter.txt

```diff
@@ -47,6 +47,283 @@ Set Query Filter
 
    .. include:: /includes/extracts/query-bar-results.rst
 
+Examples
+--------
+
+The examples on this page use a small example dataset. To import the
+sample data into your MongoDB deployment, perform the following steps:
+
+1. Copy the following documents to your clipboard:
```

Comment:
[question]

I'm curious if there's any benefit to using a `procedure::` directive as opposed to using an ordered list here. 

Either way seems reasonable, I just wanted to note it.

Reply:
Good idea. I'll update to use the procedure directive.

Reply:
Ah, so using the procedure directive caused the step titles to appear in the right-hand toc, which results in a confusing / messy presentation.

I think if the steps didn't appear in the toc, the procedure directive would improve the presentation of the main page, but I don't think it works well in this case. Thanks for the suggestion!

Reply:
Got it -- thanks for the info!